tensorflow/lite/experimental/micro/kernels/BUILD

```diff
@@ -50,6 +51,7 @@ cc_library(
     name = "portable_optimized_micro_ops",
     srcs = [
         "fully_connected.cc",
+        "pooling.cc",
```

Comment:
Do we really have an optimized version already?

Reply:
No, that line made it in by mistake, I'll remove it.